jsx/Form.js

```diff
@@ -538,7 +538,7 @@ SelectElement.defaultProps = {
   multiple: false,
   disabled: false,
   required: false,
-  sortByValue: true,
+  sortByValue: false,
```

Comment:
I don't think this default should be changed. Sorting a sensible default. Changing this PR so it's just the issue tracker priority prop that's changed would likely fix Travis.

Reply:
There's [a number of other places](https://github.com/aces/Loris/issues/6328#issuecomment-610599920) (at least) where the sort-order is having an unintended impact.  
Note -- given we limit this PR to the Issue tracker priority, tickets should be created to address these other sorting dropdowns individually.  And should be documented in our "how to build a module" docs

Reply:
I think ti would be a good idea to make those tickets. They would make for good Beginner/GSOC tickets.

Reply:
I agree with Dave